diesel_codegen/src/schema_inference/pg.rs

```diff
@@ -38,11 +40,11 @@ table! {
 }
 
 pub fn determine_column_type(cx: &mut ExtCtxt, attr: &ColumnInformation) -> P<ast::Ty> {
-    let tpe = if attr.type_name.starts_with("_") {
-        let subtype = str_to_ident(&capitalize(&attr.type_name[1..]));
+    let tpe = if attr.type_name.ends_with("[]") {
```

Comment:
I've changed the query to get the column informations. Resulting of this the type names changed a bit. One of this changes results in a rename of the array type names. They don't start with _ anymore, but they seems to end with [] now.
Check this query
```sql
select oid::regtype, typname from pg_type where typname like '\_%';
```